Sources/AblyChat/Room.swift

```diff
@@ -61,13 +64,23 @@ internal actor DefaultRoom: Room {
             throw ARTErrorInfo.create(withCode: 40000, message: "Ensure your Realtime instance is initialized with a clientId.")
         }
 
+        channels = Self.createChannels(roomID: roomID, realtime: realtime)
+
         messages = await DefaultMessages(
+            channel: channels[.messages]!,
             chatAPI: chatAPI,
             roomID: roomID,
             clientID: clientId
```

Comment:
I know `channels` is created within the `init` so the `.messages` channel is always there but it's still a code smell to force unwrap imo. 

We don't yet have a best practices per say, but I'd be keen to avoid force-unwrapping in any user-facing/production code unless for a very specific, documented exception. I think it's fine in test libraries, just not the actual framework

Reply:
@umair-ably that's fair — we need to revisit this code anyway (see https://github.com/ably-labs/ably-chat-swift/issues/105; I’ve mentioned your point there)

Reply:
Will resolve this one